ReactAndroid/src/main/java/com/facebook/react/modules/image/ImageLoaderModule.java

```diff
@@ -45,6 +48,48 @@ public String getName() {
     return "ImageLoader";
   }
 
+  @ReactMethod
+  public void getSize(
+      String uriString,
+      final Callback successCallback,
```

Comment:
The API currently does not use a Promise in iOS, doesn't it mean that if I change that to a Promise that it will be incompatible?